docs/guides/airnode/calling-an-airnode/index.md

```diff
@@ -174,6 +192,18 @@ function in the code sample below.
   been encoded and contains a timestamp in addition to other response data.
   Decode it using the function decode() from the abi object.
 
+### `withdraw()` method
```

Comment:
Perhaps "Withdrawal" or something similar given the other h3 headings in this section are "Request Parameters" and "Response Parameters"

Reply:
Yeah, I agree. Fixed this too 👍🏻